src/features/recruitments/components/tech-stack-field.tsx

```
@@ -0,0 +1,98 @@
+
+          {/* 우: 스택 체크리스트 */}
+          <ul className="no-scrollbar grid max-h-60 flex-1 grid-cols-2 content-start overflow-y-auto py-2">
+            {stacks.map((stack) => (
+              <li key={stack}>
+                <label className="hover:bg-secondary flex cursor-pointer items-center gap-2.5 rounded-md px-3 py-2 text-sm">
+                  <Checkbox
+                    checked={selected.includes(stack)}
+                    onCheckedChange={() => toggle(stack)}
+                  />
+                  <span>{stack}</span>
+                </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)